web/embed_style.html

```diff
@@ -56,7 +56,7 @@
 
   #leftPanel {
     width: 60%;
-    /*height: 100%;*/
+    height: 100%;
```

Comment:
Can you test this on the deployed version? This height setting had been causing the console view to position itself partly off the screen.